sinfonier-web/app/models/module/Module.java

```diff
@@ -273,6 +274,13 @@ public static ModulesContainer find(ModuleSearch search, User user, Integer page
     if (search.getType() != null && search.getType().length() > 0) {
       query.add(new BasicDBObject(FIELD_TYPE, search.getType()));
     }
+    
+    if (search.getDescription() != null && search.getDescription().length() > 0) {
+      BasicDBList list = new BasicDBList();
+      list.add(new BasicDBObject(FIELD_VERSIONS+"."+FIELD_VERSION_ID, new BasicDBObject("$in", getModuleVersionIdsByDescription(search.getDescription()))));

```

Comment:
In my local database I have most version ids into modules like simple Strings, but several version ids like ObjectIds. I need this double check to find all modules with a description on their version